app.go

```diff
@@ -114,14 +113,23 @@ func main() {
 	_ = json.Unmarshal([]byte(byteValue), &pd.Rulebar)
 
 	log.Println("Server Startup at: localhost" + pd.Port)
-	nosql.MongoInit()
-	defer nosql.MongoKill()
+
+	// Attempt to connect to MongoDB with a 30 second timeout
+	// err = nosql.MongoInit(time.Second * 30)
+	// if err != nil {
+	// 	log.Printf("Could not connect the Mongo client: err = %s", err)
+	// 	os.Exit(1)
+	// }
```

Comment:
It'd be good to either delete this or leave a comment w/ a ticket or something explains why it's commented out. 

In the context of this PR, my guess is because it saves from having to bother with a mongo service in docker-compose for a rule we don't support anyway? This is fine, but in a few months that context is gonna be lost without looking at git history. Maybe we can leave it in place, but change it to something like:
```
go func() {
    if err := nosql.MongoInit(time.Second * 30); err == nil {
       // defer ...
       // http.HandleFunc ...
    } else {
        log.Printf("could not connect to mongo service:", err)
    }
}()
```

Reply:
This will be added back in future changes and commenting it out allows for the UI to provide the default "under construction" ui. Harold asked about tickets I'm waiting for his response on that.